### ISSUE TITLE
[BUG] Can't add to cart if "Prevent Redicrect" option enabled

### ISSUE THREAD

**Issue by rohanrichards** [IsThereAnyDeal/AugmentedSteam#1863]
### Expected Behavior
Items would add to cart after clicking "add to cart".
Can move through wishlist or store adding multiple items.
Can navigate to cart and see all items.

### Actual Behavior
Can click button to add items, button changes to "In Cart"
Navigating to cart shows an empty cart

### Steps to Reproduce the Problem

  1. Enable "Prevent redirection when adding items to your Cart"
  2. Add items to cart
  3. Visit cart

### Notes
If you disable the option an add an item to the cart, steam shows this new(ish) dialogue that is being blocked by the option. If this dialogue is not allowed/clicked then the item doesn't make it onto the cart
![image](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/16222002/9b80b202-d650-43ef-aa53-1ff126dd09a5)

### Specifications

  - Augmented Steam Version: 3.0.0
  - Browser Version: Firefox 123.0.1 (64-bit)

**Comment by candela97** [comment]
Duplicate of #1851
